typescript/packages/lookslike-high-level/src/recipes/fetchExample.ts

```diff
@@ -0,0 +1,37 @@
+import { html } from "@commontools/common-html";
+import { recipe, fetchData, UI, NAME, ifElse } from "@commontools/common-builder";
+
+interface Item {
+  id: string;
+  prompt: string;
+}
+
+export const fetchExample = recipe<{ url: string }>("Fetch Example", ({ url }) => {
+  const { result } = fetchData<{ result: Item[] }>({
+    url,
+    schema: {
+      type: "array",
+      items: {
+        type: "object",
+        properties: {
+          id: { type: "string" },
+          prompt: { type: "string" },
+        },
+      },
+    },
+  });
+
+  return {
+    [NAME]: "Fetch Example",
+    [UI]: html`<div>
+      ${ifElse(
+        result,
+        html`<div>${
+            result.map(({ prompt, id }) => html`<div>${prompt} - ${id}</div>`)
```

Comment:
not sure what I did different than generateData to get this ts error:

```
ts: Property 'map' does not exist on type 'Value<{ result: Item[]; }>'.
  Property 'map' does not exist on type '{ result: Value<Item[]>; }'.
```

Reply:
is this still not working?